charts/milvus/values.yaml

```diff
@@ -901,6 +901,7 @@ externalPulsar:
 ###################################
 # External kafka
 # - these configs are only used when `externalKafka.enabled` is true
+# - note that the following are just examples, you should confirm the value of brokerList and mechanisms according to the actual external Kafka configuration
```

Comment:
We'd better add an example for msk with sasl_ssl in the comment. And for the readability, we should split long comments into separate lines.